### PR TITLE
[easy] Add testing utilties for torch.nn.utils.set_swap_module_params_on_conversion

### PR DESCRIPTION
**Pull request by mikaylagawarecki**
For above PR to parametrize existing `load_state_dict` tests

Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #118023
* #117167
* #118028

### CODE REVIEW

File: test/test_modules.py [hunk 1]
```diff
@@ -866,65 +867,62 @@ def test_to(self, device, dtype, module_info, training, swap, set_grad):
                                                        requires_grad=False, training=training)
         torch.nn.utils.set_swap_module_params_on_conversion(swap)
 
-        try:
```
Comment (mikaylagawarecki) [inline test/test_modules.py:869]: the diff is messed up here, all I did was remove the `try: finally` block

File: test/test_modules.py [hunk 2]
```diff
@@ -856,6 +856,7 @@ def test_errors(self, device, dtype, module_info, training):
     @modules([module for module in module_db if not module.is_lazy])
     @parametrize('swap', [True, False])
     @parametrize('set_grad', [True, False])
+    @wrapSwapTensorsTest(swap=None)
```
Comment (albanD) [inline test/test_modules.py:859]: You don't need to pass in the swap argument here right?

**Review by albanD — APPROVED** [review]

**Pull request closed without merging**